assets/js/default-calendar.js

```
@@ -337,45 +337,55 @@ jQuery(function ($) {
-									last.hide();
-								}
-							}
-							last = current;
-						},
+					  render: function (event, api) {
+						setTimeout(() => {
+						  api.reposition();
+						}, 10); // ensures top/left are recalculated after DOM draw
+					  },
+					  show: function (event, current) {
+						if (last && last.id && last.id !== current.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)